scripts/provisioning/roles/motr-build/tasks/isa-l.spec

```diff
@@ -0,0 +1,50 @@
+%undefine _disable_source_fetch
+
+Name:		isa-l
+Version:	%{_isa_l_version}
+Release:	1%{?dist}
+Summary:	Intel(R) Intelligent Storage Acceleration Library.
+
+Group:		Applications
+License:	BSD-3-Clause
+URL:		https://github.com/intel/isa-l/
+Source0:	https://github.com/intel/isa-l/archive/v%{_isa_l_version}.tar.gz
+
+BuildRequires:	autoconf libtool yasm make
```

Comment:
"yasm" should be added to the dependency list, just like we did for nasm or other tools.

Reply:
Done